Auth/AuthAPI.py

```diff
@@ -1,6 +1,38 @@
-from flask import Blueprint
+from flask import Blueprint, request
 import sys
-from db import db
+from flask_cors import cross_origin
+import jwt
+
 sys.path.append("../")
 
-auth_api = Blueprint("auth", __name__)
\ No newline at end of file
+auth_api = Blueprint("auth", __name__)
+
+local_array = {}
+
+
+@auth_api.route("/register", methods=["POST"])
+@cross_origin(supports_credentials=True)
+def register_user():
+    try:
+        username = request.args.get('username')
+        password = request.args.get('password')
+        local_array[username] = password
+        return {'status': 'success'}
+    except Exception as e:
+        return {"err": "An error has occured", "status": " failed"}, 500
+
+
+@auth_api.route("/login", methods=["POST"])
+@cross_origin(supports_credentials=True)
+def login_user():
+    try:
+        username = request.args.get('username')
+        password_hash = request.args.get('passwordHash')
```

Comment:
line 18 is password but this line is passwordHash. after registering the user (with username and password), when i do a login, the user doesnt have a passwordHash. consider changing line 18 to `get('passwordHash')`, or this line to `get('password')`